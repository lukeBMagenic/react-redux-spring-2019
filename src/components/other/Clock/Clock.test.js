import React from 'react';
import { shallow } from "enzyme";

import Clock from './Clock';

describe("Clock", () => {
  it("should render correctly", () => {
    const component = shallow(<Clock />);

    //expect(component).toMatchSnapshot();
  });
});