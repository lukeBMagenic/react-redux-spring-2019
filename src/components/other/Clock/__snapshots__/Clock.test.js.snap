// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clock should render correctly 1`] = `
<div>
  <h2>
    It is 
    11:56:51 AM
    .
  </h2>
</div>
`;
